plugins/kubernetes-react/config.d.ts

```diff
@@ -27,5 +27,25 @@ export interface Config {
        */
       enabled?: boolean;
     };
+    /**
+     * Frontend config
+     */
+    frontend?: {
+      /**
+       * Pod Delete config
+       */
+      podDelete?: {
+        /**
+         * Enable `podDelete` UI feature
+         * @visibility frontend
+         */
+        enabled?: boolean;
+        /**
+         * Override the `podDelete` button text
+         * @visibility frontend
+         */
+        buttonText?: string;
```

Comment:
Could you use the i18n APIs for this instead? It covers this use-case and more https://backstage.io/docs/plugins/internationalization

Reply:
besides this `buttonText` key is to give it a different text from the default one, Restart pod over Delete pod for example, wasn't meant to be used as a translation.

Reply:
OK, what if I remove the key in the documentation so that it's hidden and we merge this PR?
In this way, we can start to drop the k8 plugins internally in the company and then I'll work on adding the i18n API, or drop this key and use a single text for the button.

If in my company they are OK with using delete pod over restart pod what are your thoughts on this key? still useful or do we drop it?